Volume/Views/Onboarding/OnboardingView.swift

```diff
@@ -7,11 +7,13 @@
 //
 
 import AppDevAnalytics
+import Combine
 import SwiftUI
 
 struct OnboardingView: View {
     @State private var isShowingSplash = true
     @State private var page: Page = .welcome
+    @State private var cancellableCreateUserMutation: AnyCancellable?
```

Comment:
I think our convention in the app is to just call this `cancellable`, though I'm not sure. If you want to stick with this, I would call it `createUserMutationCancellable` or `createUserCancellable` to be more concise